### ISSUE TITLE
Firefox SSL authentification

### ISSUE THREAD

**Issue by aarnaud** [aarnaud/lxd-webui#10]
@saldru report this issues https://github.com/dobin/lxd-webgui/issues/5#issuecomment-221000729

maybe: http://stackoverflow.com/questions/24005354/ssl-client-certificate-not-loaded-on-ajax-call-from-firefox

**Comment by saldru** [comment]
@aarnaud Yes, I haven't test with Google Chrome, Internet Explorer, Edge or Opera, just with Firefox.
I will test with other Web Browsers.

**Comment by saldru** [comment]
@aarnaud Yes, Firefox it's the problem, I have tried with Google Chrome and It's working.

**Comment by aarnaud** [comment]
@saldru I will fix when angular 2 fix https://github.com/angular/angular/pull/7281

Your client brower is under Windows, Linux or Mac ? I can package an desktop applications with electron.

**Comment by saldru** [comment]
@aarnaud I have tried with Firefox 46.0.1 (x64) for Windows and Iceweasel 38.7.1 (x86) for Debian Jessie and I have the same error. 
My default Web Browser is under Windows.

**Comment by lcherone** [comment]
I use https://lxc.systems to control multiple LXD servers, you **dont need to add browser certificates**, works in all browsers, plus it manages nginx and iptables rules, just how it should be...

**Comment by aarnaud** [comment]
@lcherone  Great works on https://lxc.systems but it's not an opensource project. 
@saldru If you don't want SSL Cert Authentication in your browser, you can add a reverse-proxy nginx with this authentication. 
lxd-webui don't provide anything server side

**Comment by Shark-Linux** [comment]
> If you don't want SSL Cert Authentication in your browser, you can add a reverse-proxy nginx with this authentication.

@aarnaud Any advice on exactly how to correctly configure that? Looking to automate a setup and the browser/certs is the only obstacle that remains.
